src/qt/transactiontablemodel.cpp

```diff
@@ -96,25 +96,29 @@ class TransactionTablePriv
      */
     QList<TransactionRecord> cachedWallet;
 
-    bool fQueueNotifications = false;
+    /** True when model finishes loading all wallet transactions on start */
+    bool m_loaded = false;
+    /** True when transactions are being notified, for instance when scanning */
```

Comment:
It could be just my bad English, but it seems should be "... are _not_ being notified", no?

If I'm correct, maybe re-word the comment in positive one?

Or maybe we think about different notifications: (a) from the node vs (b) to a user

Reply:
Right, I'll rephrase.

Reply:
In commit "qt: Refactor ShowProgress to DispatchNotifications" (c6cbdf1a90a253fef0259b365a782bf88cd437f2)

re: https://github.com/bitcoin-core/gui/pull/121#discussion_r623855013

> Right, I'll rephrase.

If you're still planning to update this, my attempt at a comment would be: "This will be true when blocks in the chain are being scanned for new transactions, and transaction notification popups should be suppressed until the scan is done. It can be true even after m_loaded is true if a rescan has been triggered." I think I would also call this `m_delay_notifications` instead of `m_loading` because it can be true when `m_loaded` is also true, but feel free to stick with whatever name you prefer.